pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -92,6 +92,8 @@ self: super: {
         rm -r $out/doc/?ndroid*
       '';
     };
+
+    passthru.shellPath = "/bin/git-annex-shell";
```

Comment:
Could you add a short comment here explaining this?  (Basically just what you said in the body of your PR)

Reply:
Sure, one sec.

Reply:
@cdepillabout donezo! :)